pkg/tests/api/alerting/api_prometheus_test.go

```diff
@@ -671,58 +672,16 @@ func TestPrometheusRulesPermissions(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, 200, resp.StatusCode)
 
-		require.JSONEq(t, `
-{
-	"status": "success",
-	"data": {
-		"groups": [{
-			"name": "arulegroup",
-			"file": "folder1",
-			"rules": [{
-				"state": "inactive",
-				"name": "rule under folder folder1",
-				"query": "[{\"refId\":\"A\",\"queryType\":\"\",\"relativeTimeRange\":{\"from\":18000,\"to\":10800},\"datasourceUid\":\"-100\",\"model\":{\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":43200,\"type\":\"math\"}}]",
-				"duration": 120,
-				"annotations": {
-					"annotation1": "val1"
-				},
-				"labels": {
-					"label1": "val1"
-				},
-				"health": "ok",
-				"type": "alerting",
-				"lastEvaluation": "0001-01-01T00:00:00Z",
-				"evaluationTime": 0
-			}],
-			"interval": 60,
-			"lastEvaluation": "0001-01-01T00:00:00Z",
-			"evaluationTime": 0
-		},
-		{
-			"name": "arulegroup",
-			"file": "folder2",
-			"rules": [{
-				"state": "inactive",
-				"name": "rule under folder folder2",
-				"query": "[{\"refId\":\"A\",\"queryType\":\"\",\"relativeTimeRange\":{\"from\":18000,\"to\":10800},\"datasourceUid\":\"-100\",\"model\":{\"expression\":\"2 + 3 \\u003e 1\",\"intervalMs\":1000,\"maxDataPoints\":43200,\"type\":\"math\"}}]",
-				"duration": 120,
-				"annotations": {
-					"annotation1": "val1"
-				},
-				"labels": {
-					"label1": "val1"
-				},
-				"health": "ok",
-				"type": "alerting",
-				"lastEvaluation": "0001-01-01T00:00:00Z",
-				"evaluationTime": 0
-			}],
-			"interval": 60,
-			"lastEvaluation": "0001-01-01T00:00:00Z",
-			"evaluationTime": 0
-		}]
-	}
-}`, string(b))
+		body := asJson(t, b)
```

Comment:
There are a couple other tests in this file which assert in the same way. If we like this, I can propagate it to the other tests too.